HolaMundo/.editorconfig

```
@@ -0,0 +1,17 @@
+# EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+
+# We recommend you to keep these unchanged
+end_of_line = lf
+charset = utf-8
```

```suggestion
charset = utf-16
```
¿Qué opinas de utilizar utf-16 en vez de 8? Ese es el que usa Windows

---

No tiene sentido ese cambio

---

Estoy de acuerdo